hip/static/styles/pages/base.scss

```diff
@@ -1,5 +1,13 @@
 @import "../colors.scss";
```

Comment:
Seems like sometimes we import without the `.scss` suffix and sometimes we import with it. I'm assuming that both work, but maybe we should stick with one? I think i'd slightly prefer including the suffix, to be more explicit, but am fine with either choice.

(Comment applies to multiple files, not just this location)